crates/ra_assists/src/handlers/merge_match_arms.rs

```diff
@@ -81,10 +81,7 @@ pub(crate) fn merge_match_arms(acc: &mut Assists, ctx: &AssistContext) -> Option
 }
 
 fn contains_placeholder(a: &ast::MatchArm) -> bool {
-    match a.pat() {
-        Some(ra_syntax::ast::Pat::PlaceholderPat(..)) => true,
-        _ => false,
-    }
+    matches!(a.pat(), Some(ra_syntax::ast::Pat::PlaceholderPat(..)))
```

Comment:
Let’s import ast while we are at it.

Reply:
Done https://github.com/rust-analyzer/rust-analyzer/pull/5105/commits/503f9498cd469c6467250c02bd8f6dbce973dd57